src/environment-data-provider.ts

```diff
@@ -9,13 +9,24 @@ export function getEnvironmentDataFromResults(
     return {
         timestamp: axeResults.timestamp,
         targetPageUrl: axeResults.url,
+        axeVersion: axeResults.testEngine.version,
     };
 }
 
 export function getEnvironmentDataFromEnvironment(): EnvironmentData {
+    // We use the global axe rather than the imported axe to match the version
+    // we're scanning from the context of
```

Comment:
I'm a little lost in translation here... it seems this phrase got cut of? Maybe is just me